routes/api.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Http\Request;
+
+/*
+|--------------------------------------------------------------------------
+| API Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register API routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| is assigned the "api" middleware group. Enjoy building your API!
+|
+*/
+
+Route::middleware('auth:api')->get('/user', function (Request $request) {
+    return $request->user();
+});
+
+Route::apiResource('ongs', 'API\OngController');
+Route::apiResource('sessions', 'API\SessionController');
+Route::apiResource('profile', 'API\ProfileController');
+Route::apiResource('incidents', 'API\IncidentsController');
```

Comment:
O namespace Api\ pode ser externalizado usando grupo de rotas.
O grupo permite aplicar alteracoes à todas as rotas. Supondo que você queira adicionar um prefixo na url para /api/v1/, basta alterar no grupo (Ex.: prefixo para v1), aplicar middlewares, etc.

```suggestion
Route::namespace('API')->group(function () {
    Route::apiResource('ongs', 'OngController');
    Route::apiResource('sessions', 'SessionController');
    Route::apiResource('profile', 'ProfileController');
    Route::apiResource('incidents', 'IncidentsController');
});
```